administrator/language/de-DE/plg_user_token.ini

```
@@ -0,0 +1,24 @@
+; Joomla! German Translation
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; Copyright (C) Translation 2008 - 2019 J!German (www.jgerman.de)
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_USER_TOKEN="Benutzer - Joomla Token"
+PLG_USER_TOKEN_XML_DESCRIPTION="Ermöglicht die Verwaltung von Sicherheitstoken, die für die Authentifizierung an der Joomla-API-Anwendung (Remote-Zugriff auf Ihre Website) verwendet werden. Diese Token sind streng persönlich. Man kann sich nur die eigenen Token anzeigen, auch Super Benutzer können nur die Token anderer Benutzer deaktivieren oder zurücksetzen."
```

falls nur 1 Token / Benutzer

```suggestion
PLG_USER_TOKEN_XML_DESCRIPTION="Ermöglicht die Verwaltung von Sicherheitstoken, die für die Authentifizierung an der Joomla-API-Anwendung (Remote-Zugriff auf die Website) verwendet werden. Diese Token sind streng vertraulich/persönlich. Es kann nur das eigene Token angezeigt werden. Super Benutzer können Token anderer Benutzer nur aktivieren, deaktivieren oder zurücksetzen."
```